.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*]
+indent_style = space
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_size = 2
+
+[*.md]
+indent_size = 4
+trim_trailing_whitespace = false
```

Comment:
add another line at the end of the file this is known as EOF
you'll notice this when you see that red circle with a dash in the middle